runtime/api/Gamepads.py

```diff
@@ -51,18 +51,132 @@ def get_joysticks(index):
     """
     return mc.get('gamepad')[index]['axes']
 
-def get_buttons(index):
+def get_axis(index,axis):
+    """Returns the position of a specified joystick.
+
+    Each returned value is between -1 and 1, which represents where the joystick
+    is along that axis. In order to get a better sense of
+    the joystick mappings, click the 'Details' button next to a gamepad in Dawn
+    or refer to https://w3c.github.io/gamepad/#remapping.
+
+    On a standard gamepad:
+    - axes[0] represents the horizontal axis of the left joystick
```

Comment:
You should create an enum that maps from meaningful names (e.g. AXIS_LEFT_X, AXIS_LEFT_Y, and the like) and axis indices